docs/site/Deploying_to_ibm_cloud_kubernetes.md

```diff
@@ -45,7 +45,7 @@ $ lb4 app
 ? Project root directory: deploy-test
 ? Application class name: DeployTestApplication
 ? Select features to enable in the project Enable tslint, Enable prettier, Enable mocha, Enable loopback
-Build, Enable vscode, Enable repositories, Enable services
+Build, Enable vscode, Disable docker, Enable repositories, Enable services
```

Comment:
We probably should update the tutorial to leverage `--docker` for `lb4 app` and IBM cloud's ability to deploy the CloudFoundry application from the docker image:
```sh
lb4 my-lb4-app
cd my-lb4-app
npm run docker:build
docker push ...
...
ibmcloud cf push my-lb4-app --docker-image=<username>/my-lb4-app:latest
```
See https://github.com/IBM-Cloud/logistics-wizard/blob/master/Deploy_Microservices_Cloud_Foundry_Docker.md

Reply:
@nabdelgadir , Raymond decided he wants 'Enable Docker' set to true as default.

Reply:
I see that you've updated it in your [PR](https://github.com/strongloop/loopback-next/pull/2539) so I'll remove it from this one.